### DISCUSSION TITLE
Bomb Defusal Contract

### DISCUSSION THREAD

**Discussion by TheBeastLegend** [Malcolm-Q/LC-LateGameUpgrades#236] [Q&A] [answered]
Bought the contract for 500 and defusing the bomb only gives 500, is this intended? Seems very not worth.

**Comment by Malcolm-Q** [comment, marked as answer]
The original intention for contracts was to give players additional means to achieve higher quotas. If you do one contract a day you're getting an extra $1500 towards your quota.

However like the rest of the mod both the price and reward and more regarding contracts is configurable so you can have your own desired behaviour.

Additionally alternate configurable behaviours for how contracts can effect your game will probably be coming soon. (Percentage based rewards, incrementing costs, etc)
